Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -495,11 +495,12 @@ div.bocList div.bocListTableBlock div.bocListTableContainer div.bocListFakeTable
   overflow: hidden;
 }
 
+
 div.bocList div.bocListTableBlock div.bocListTableContainer div.bocListFakeTableHead > div
 {
   /* Reserve space in case there is a scrollbar for the data-part of the table.
-     Note that the actual max-width is not important so long as the scrollbar's max-width is covered. '*/
-  border-right: var(--remotion-themed-scrollbar-width) solid var(--boclist-tablehead-background-color);
+ that the actual max-width is not important so long as the scrollbar's max-width is covered. '*/
+  padding-right: 10px;
```

Comment:
This should be declared as a css-variable together with the existing `--remotion-themed-scrollbar-style:` property to make it clear what requires changing when the width is manipulated. The intention of the css-variables to not have magic numbers in code or duplicated business logic.

And even though we do not use it with `--remotion-themed-popup-scrollbar-style` in remotion, we probably should prepare that property, too, in case someone needs this informaton.